react-app/src/pages/Forbidden.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Paper, Typography } from '@mui/material';
+import React from 'react';
+
+const Forbidden = () => {
```

Comment:
Would be good for redirections through the AuthGuard.

Reply:
Comitted this by accident... A leftover from when I was trying to do exactly that. We probably actually will want this eventually so I guess I won't get rid of it.